spec/features/draws/draw_student_report_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Draw student report' do
```

Comment:
Yes. I'm trying to figure out the best way to extract the no-longer-relevant parts of this branch without losing them entirely.